pkg/apis/eventing/v1beta1/eventtype_conversion.go

```diff
@@ -18,17 +18,55 @@ package v1beta1
 
 import (
 	"context"
-	"fmt"
+
+	"knative.dev/eventing/pkg/apis/eventing/v1beta2"
 
 	"knative.dev/pkg/apis"
 )
 
 // ConvertTo implements apis.Convertible
-func (source *EventType) ConvertTo(ctx context.Context, to apis.Convertible) error {
-	return fmt.Errorf("v1beta1 is the highest known version, got: %T", to)
+func (source *EventType) ConvertTo(ctx context.Context, obj apis.Convertible) error {
+	switch sink := obj.(type) {
```

Comment:
nit: Isn't the name `sink` a bit misleading in our case? (instead of using `to`/`from`).

Reply:
@creydr I followed the pattern from previous iterations, see: https://github.com/knative/eventing/blob/release-0.20/pkg/apis/eventing/v1beta1/broker_conversion.go#L31